src/components/CloudHeader/_cloud-header.scss

```diff
@@ -0,0 +1,119 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
+$cloud-header-link-color: $inverse-01 !default;
+$cloud-header-icon-color: $inverse-01 !default;
+$cloud-header-menu-color: $brand-02 !default;
+$cloud-header-background-color: $color__blue-90 !default;
+
+.cloud-header {
```

Comment:
Should our prefix be `cloud` here, or should we follow in step with IBM/Type or IBM/Grid repos and use the `ibm` prefix?

Reply:
I'll change these to `bx--` until we make a sweeping change across all of our repos